app/models/metric_definition.rb

```diff
@@ -0,0 +1,25 @@
+# == Schema Information
+#
+# Table name: metric_definitions
+#
+#  id          :bigint           not null, primary key
+#  code        :enum             not null
+#  explanation :string           not null
+#  name        :string           not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+
+class MetricDefinition < ApplicationRecord
+  enum code: { review_turnaround: 'review_turnaround',
+               blog_visits: 'blog_visits',
+               merge_time: 'merge_time',
+               blog_post_count: 'blog_post_count',
+               open_source_visits: 'open_source_visits',
+               defect_escape_rate: 'defect_escape_rate',
+               pull_request_size: 'pull_request_size' }
+
+  validates :code, inclusion: { in: codes.keys }, uniqueness: true
```

Comment:
I think the inclusion is not necessary because it's an enum

Reply:
You're right! I'll remove it.